dev_guide/deployments.adoc

```diff
@@ -76,14 +74,21 @@ spec:
           name: "origin-ruby-sample:latest"
   strategy: <5>
     type: "Rolling"
+  paused: false <6>
+  revisionHistoryLimit: 2 <7>
+  minReadySeconds: 0 <8>
+
 ----
 
-<1> The replication controller template named `frontend` describes a simple Ruby application.
-<2> There will be 5 replicas of `frontend` by default.
-<3> A link:#config-change-trigger[configuration change trigger] causes a new deployment to be created any time the replication controller template changes.
+<1> The pod template of the `frontend` deployment configuration describes a simple Ruby application.
+<2> There will be 5 replicas of `frontend`.
+<3> A link:#config-change-trigger[configuration change trigger] causes a new deployment to be created any time the pod template changes.
```

Comment:
We just converted to xrefs, so we will now only be using link format for external links.
s/link:#config-change-trigger[configuration change trigger]/xref:config-change-trigger[configuration change trigger]